### PR TITLE
ref #3241 upgrade for controlpanel error-log-form view link

### PR DESCRIPTION
**Pull request by jmevissen**
* upgrade step to use renamed error-log-form view link in ControlPanel

### CODE REVIEW

File: plone/app/upgrade/v60/profiles.zcml [hunk 1]
```diff
@@ -11,6 +11,14 @@
       provides="Products.GenericSetup.interfaces.EXTENSION"
       />
 
+  <genericsetup:registerProfile
+      name="to6003"
+      title="Upgrade profile for Plone 5.2.x to Plone 6.0.0a3.dev0 (6003)"
```
Comment (mauritsvanrees) [inline plone/app/upgrade/v60/profiles.zcml:16]: I would simplify the title to:

"Upgrade profile to Plone 6.0.0a3 (6003)"

File: plone/app/upgrade/v60/profiles/to6003/controlpanel.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<object name="portal_controlpanel" meta_type="Plone Control Panel Tool"
+  i18n:domain="plone" xmlns:i18n="http://xml.zope.org/namespaces/i18n">
+
+  <!-- update -->
+  <configlet title="Errors" action_id="errorLog" appId="ErrorLog"
+    category="plone-security" condition_expr=""
+    icon_expr="string:exclamation-square"
+    url_expr="string:${portal_url}/@@error-log-form" visible="True"
```
Comment (mauritsvanrees) [inline plone/app/upgrade/v60/profiles/to6003/controlpanel.xml:9]: I would only include the `appId`, `category`, and the new `url_expr` here, and remove the other attributes.

Comment (jmevissen) [reply plone/app/upgrade/v60/profiles/to6003/controlpanel.xml:9]: Hy, I tried leaving out unchanged attributes. When leaving out  visible="True" the ErrorLogForm link is not displayed in the ControlPanel after running the Upgrade. Maybe it is better to use the whole configuration?

Comment (mauritsvanrees) [reply plone/app/upgrade/v60/profiles/to6003/controlpanel.xml:9]: I checked, and you are right. Basically every attribute in here is required. Sample traceback with `title` attribute missing:

```
Traceback (innermost last):
  Module ZPublisher.WSGIPublisher, line 167, in transaction_pubevents
  Module ZPublisher.WSGIPublisher, line 376, in publish_module
  Module ZPublisher.WSGIPublisher, line 271, in publish
  Module ZPublisher.mapply, line 85, in mapply
  Module ZPublisher.WSGIPublisher, line 68, in call_object
  Module Products.GenericSetup.tool, line 1106, in manage_doUpgrades
  Module Products.GenericSetup.upgrade, line 202, in doStep
  Module Products.GenericSetup.tool, line 399, in runAllImportStepsFromProfile
   - __traceback_info__: profile-plone.app.upgrade.v60:to6003
  Module Products.GenericSetup.tool, line 1487, in _runImportStepsFromContext
  Module Products.GenericSetup.tool, line 1299, in _doRunImportStep
   - __traceback_info__: controlpanel
  Module Products.CMFPlone.exportimport.controlpanel, line 165, in importControlPanel
  Module Products.GenericSetup.utils, line 957, in importObjects
   - __traceback_info__: portal_controlpanel
  Module Products.GenericSetup.utils, line 558, in _importBody
  Module Products.CMFPlone.exportimport.controlpanel, line 55, in _importNode
  Module Products.CMFPlone.exportimport.controlpanel, line 64, in _initProvider
  Module Products.CMFPlone.exportimport.controlpanel, line 146, in _initConfiglets
  Module Products.CMFPlone.PloneControlPanel, line 229, in addAction
ValueError: A name is required.
```

A few can be left out, but then existing values will get overwritten with a default. So your original code is good.
We should probably improve the import handler at some point.

**Review by mauritsvanrees — APPROVED** [review]
LGTM.

@jenkins-plone-org please run jobs

**Pull request merged**